tox.ini

```diff
@@ -12,6 +12,9 @@ setenv = PYTHONPATH = {toxinidir}
 install_command = pip install --process-dependency-links {opts} {packages}
 commands = py.test --cov --cov-report=
 deps =
+    crccheck
```

Comment:
Why do these need to be added here? My understanding is that this list is just for extra test-time dependencies -- tox should be running a pip install of bellows after installing these. I'd prefer to keep the information about what bellows itself depends on in only one place.

Reply:
Yes, you are right. I have removed them again. I think I added them while I was testing.